lib/players/human.rb

```diff
@@ -0,0 +1,9 @@
+class Human < Player
+
+  def move(board)
+    puts "Enter a number from 1-9."
+    input = gets.strip
+    input
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Enter a number from 1-9."
  gets.strip
end
```